### DISCUSSION TITLE
Runing the Talosctl on windows?

### DISCUSSION THREAD

**Discussion by pojusoi** [siderolabs/talos#3356] [Q&A] [answered]
I'm running windows and searching support for Talosctl. Is there build and software packaging plans for windows compatible Talosctl?

**Comment by smira** [comment, marked as answer]
`talosctl` works fine under WSL. As for native support, we're looking for Windows devs to make sure `talosctl` works on Windows as expected, and we can include it into our releases.

**Reply by jbsulli** [reply]
I'm happy to help with that! I'm just getting started with Talos OS

**Reply by andrewrynhard** [reply]
Awesome, @jbsulli. Once you get around to testing that let us know!

**Reply by steverfrancis** [reply]
Talosctl  is now built for Windows and published as part of the release.
https://www.talos.dev/docs/v0.13/introduction/what-is-new/#windows-suport

**Reply by jfmatth** [reply]
would love to be included on windows testing as well.
